### PR TITLE
Enforcing linting of LESS

### PR DESCRIPTION
**Pull request by rusackas**
### CATEGORY

Choose one

- [ ] Bug Fix
- [ ] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [x] Build / Development Environment
- [ ] Documentation

### SUMMARY

Some un-prettified LESS slipped through the cracks. This PR adds Prettier check enforcement for CI, and cleans up what got through before.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by nytai — APPROVED** [review]
Noticed this too when running the `lint-fix` script. 

Proposal:
adding `prettier --check '{src,stylesheets}/**/*.{css,less,sass,scss}'` to the lint script so CI can check if `lint-fix` needs to be run.

**Comment by rusackas** [comment]
> Noticed this too when running the `lint-fix` script.
> 
> Proposal:
> adding `prettier --check '{src,stylesheets}/**/*.{css,less,sass,scss}'` to the lint script so CI can check if `lint-fix` needs to be run.

Fantastic idea... I added it to this PR :)

**Comment by mistercrunch** [comment]
I saw this too and I was wondering if `stylesheets/fonts/FiraCode/` belonged into a `vendors/` or `external/` folder instead of `stylesheets/`

It depends on whether we're taking ownership of the code/assets and are likely to alter the source, or whether we're more likely to leave it untouched or upgrade in place.

**Comment by rusackas** [comment]
> I saw this too and I was wondering if `stylesheets/fonts/FiraCode/` belonged into a `vendors/` or `external/` folder instead of `stylesheets/`
> 
> It depends on whether we're taking ownership of the code/assets and are likely to alter the source, or whether we're more likely to leave it untouched or upgrade in place.

I originally thought to put it into /stylesheets because there IS a stylesheet in there that's referenced. Putting that part in vendors/external didn't make so much sense to me since we have/will make edits to the file(s). 

How about in another PR, I move the binaries to some sort of vendor assets folder, and adjust the paths in the fonts' stylesheets? If you'd rather I shoehorn that in here, I certainly can :)

File: superset-frontend/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "prod": "node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js --mode=production --colors --progress",
     "build-dev": "cross-env NODE_OPTIONS=--max_old_space_size=8192 NODE_ENV=development webpack --mode=development --colors --progress",
     "build": "cross-env NODE_OPTIONS=--max_old_space_size=8192 NODE_ENV=production webpack --mode=production --colors --progress",
-    "lint": "eslint --ignore-path=.eslintignore --ext .js,.jsx,.ts,.tsx .",
+    "lint": "eslint --ignore-path=.eslintignore --ext .js,.jsx,.ts,.tsx . && prettier --check '{src,stylesheets}/**/*.{css,less,sass,scss}'",
```
Comment (etr2460) [inline superset-frontend/package.json]: does eslint enforce the prettier checks already? Maybe we should move them out of eslint and into prettier if we're running the prettier checks anyway?

Comment (rusackas) [reply superset-frontend/package.json]: Eslint Pretties(?) and checks the Prettification(?!) of of JS/JSX/TS/TSX files. Prettier itself is being used directly on LESS/CSS files a la carte. So now this verifies them in much the same way.

**Comment by rusackas** [comment]
@mistercrunch I moved the font binaries to a `/fonts` folder, and shuffled the LESS files around accordingly. Let me know if that suffices. :D

**Comment by rusackas** [comment]
> I saw this too and I was wondering if `stylesheets/fonts/FiraCode/` belonged into a `vendors/` or `external/` folder instead of `stylesheets/`
> 
> It depends on whether we're taking ownership of the code/assets and are likely to alter the source, or whether we're more likely to leave it untouched or upgrade in place.

Went with /fonts just to be a little more explicit.

**Review by mistercrunch — APPROVED** [review]

**Pull request merged**